package.json

```diff
@@ -3,13 +3,18 @@
   "version": "0.3.1",
   "description": "A library based on a Chrome-specific bookmarklet that extracts SVG nodes and accompanying styles from an HTML document and downloads them as an SVG file",
   "main": "./dist/main.js",
-  "browser": "./dist/main.js",
+  "module": "./dist/index.js",
+  "files": [
+    "dist"
+  ],
   "scripts": {
     "test": "jest",
     "test-ci": "jest --runInBand",
     "lint": "eslint ./src",
     "lint-fix": "eslint --fix ./src",
-    "build": "webpack src/index.js --mode=production",
+    "build": "rm -r dist && npm run build-prod && npm run build-transpile",
+    "build-prod": "webpack src/index.js --mode=production",
+    "build-transpile": "babel src --out-dir dist --copy-files",
```

Comment:
Two things here: 
- let's have a separate command `clean`: `rm -fr dist` (`f` is needed to avoid failures when there's no `dist` folder)
- to avoid copying `__mocks__` folder use `src/*` here